apps/omg_watcher_info/lib/omg_watcher_info/supervisor.ex

```diff
@@ -62,6 +62,8 @@ defmodule OMG.WatcherInfo.Supervisor do
 
     children = [
       {OMG.WatcherInfo.BlockApplicationConsumer, []},
+      {OMG.WatcherInfo.PendingBlockProcessor,
+       [processing_interval: Application.fetch_env!(:omg_watcher_info, :pending_block_processing_interval)]},
```

Comment:
extract the config into the configuration module (similarly to this https://github.com/omgnetwork/elixir-omg/blob/master/apps/omg_watcher/lib/omg_watcher/configuration.ex )
if it doesn't exist yet, it's a good opportunity to start this pattern